stackslib/src/chainstate/coordinator/mod.rs

```diff
@@ -2141,6 +2141,16 @@ impl<
         self.inner_handle_new_burnchain_block(&mut HashSet::new())
     }
 
+    /// Are affirmation maps active during the epoch?
```

Comment:
Good news -- they're going away in Epoch 3.0.  There's no longer a need for them because the chain can't fork.

Reply:
Yeah, I agree. But I think the code may have to stay in place for a while, and the intention here is to have a function that encapsulates the decision on whether an affirmation map check should occur.

Reply:
Ah, I meant to say that the check could return `false` for epoch 3.0.